embedchain/loaders/json.py

```diff
@@ -36,7 +36,7 @@ def load_data(json_data: Union[dict, str]) -> list[str]:
         return ["\n".join(useful_lines)]
 
 
-VALID_URL_PATTERN = "^https:\/\/[0-9A-Za-z]+(\.[0-9A-Za-z]+)*\/[0-9A-Za-z_\/]*\.json$"
+VALID_URL_PATTERN = "^(http|https):\/\/[0-9]{1,3}(\.[0-9]{1,3}){3}(:[0-9]+)?\/[0-9A-Za-z%_\-]+\.json$"
 
```

Comment:
the addition is great @juananpe 
But this url will remove support for the existing domain.
Can you please update your code so that both the regex are supported?

Reply:
Done. Sorry for the oversight.

I tested it with this script and possible correct and incorrect (the 2nd url is invalid) values.
Tell me if there are some other cases to test.

```python
import re

# Regex pattern
pattern = "^https?://(?:www\.)?(?:\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}|[a-zA-Z0-9.-]+)(?::\d+)?/(?:[^/\s]+/)*[^/\s]+\.json$"

# Test URLs
urls = [
    "http://192.168.1.140:9000/1%202%20Inflación-wav.json",
    "http:/google.com/test.json",
    "https://google.com/test.json",
    "https://www.google.com/test.json",
    "https://www.google.com/folder/subfolder/test.json",
    "https://www.google.com/folder/subfolder/folder/test.json"
]

# Checking each URL against the pattern
for url in urls:
    if re.match(pattern, url):
        print(f"Match found: {url}")
    else:
        print(f"No match: {url}")
```

Reply:
this looks good. thanks for your contribution @juananpe